src/components/Agents/InstalAgentInstruction/CLIInstallAgent.tsx

```diff
@@ -24,53 +21,21 @@ helm install mc-agent flanksource/mission-control-agent -n "mission-control-agen
 
 {{#if kubeOptions}}
 helm install mc-agent-kubernetes flanksource/mission-control-kubernetes -n "mission-control-agent"  \\
-  --set scraper.clusterName="{{agentFormValues.name}}" \\
-  --set scraper.interval="{{kubeOptions.interval}}"
+  --set clusterName="{{agentFormValues.name}}" \\
+  --set scraper.schedule="{{kubeOptions.schedule}}"
```

Comment:
these should be properties passed into a HelmChart component that then selectively renders the flux or helm cli commands. What are we doing for the registry charts ?